src/web_app_skeleton/docker/development.dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM crystallang/crystal:1.1.1
```

Comment:
Might as well use latest here imho

Reply:
Unfortunately, on M1, I ran into [some issues](https://github.com/crystal-lang/crystal/issues/11703) -- for now this is just 1.1.1 so I could get it moving forward but yes this should be updated to be correct before merge.